doc/source/api.rst

```diff
@@ -692,6 +698,10 @@ adding ordering information or special categories is need at creation time of th
 .. autosummary::
    :toctree: generated/
 
+   Categorical.categories
```

Comment:
I know that this is the wrong place for this (doesn't fit w/ the text above). I'm going to make a dedicated section for `Categorical`.